src/main/groovy/com/ullink/testtools/elastic/TestExportTask.groovy

```diff
@@ -44,6 +45,14 @@ class TestExportTask extends Exec {
     @Optional
     def targetDirectory
 
+    @Input
+    @Optional
+    def enrichment
+
+    @Input
+    @Optional
+    def featureExport
```

Comment:
we can add another task instead of export using a closure

Reply:
As in a new FeatureExportTask class inside the project that just prompts the ullink plugin to export the feature?

Reply:
a new task depends on this task, exporting the feature is ok